docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -81,6 +82,27 @@ These are the properties returned by the JSON document:
       }
   }
 
+
+The various values that may appear in the *mode* item are shown in the table below
+
++---------+---------------------------------------------------------------------------------------+
+| Mode    | Description                                                                           |
++=========+=======================================================================================+
+| poll    | The plugin is a polled plugin and *plugin_poll* will be called periodically to obtain |
+|         | new values.                                                                           |
++---------+---------------------------------------------------------------------------------------+
+| async   | The plugin is an asynchronous plugin, *plugin_poll* will not be called and the        |
+|         | plugin will be supplied with a callback function that it calls each time it has a     |
+|         | new value to pass to the Fledge system. The *plugin_register_ingest* entry point      |
+|         | will be called to register the callback with the plugin.                              |
++---------+---------------------------------------------------------------------------------------+
+| none    | This is equivalent to poll.                                                           |
++---------+---------------------------------------------------------------------------------------+
+| control | The plugin support a control flow from Fledge to the device the plugin is connected   |
+|         | to. The plugin must supply the control entry points *plugin_write* and                |
+|         | *plugin_operation*                                                                    |
```

Comment:
Table will break when it comes t white-labelled as Fledge text is used inside.
Better to use a list-table directive syntax and having defined widths and header rows